app/repositories/projects/ProjectsRepository.java

```diff
@@ -12,6 +15,8 @@
 
     Project getProjectById(final ObjectId projectId);
 
+    ObjectId createMeasurement(final ObjectId projectId, final String roomName, final MeasurementMetadata measurementMetadata);
```

Comment:
Two methods called `createMeasurement()` in ProjectsRepository and MeasurementsRepository are confusing.

Reply:
Also `createMeasurement()` but `addProject()` is not consistent.